### PR TITLE
docs: Add 'benchmark' scope to PR Title Conventions documentation

### PR DESCRIPTION
**Pull request by csuermann**
Adds 'benchmark' scope and improves some of the markdown formatting.

### CODE REVIEW

File: .github/pull_request_title_conventions.md [hunk 1]
```diff
@@ -27,26 +27,29 @@ A PR title consists of these elements:
...
+| type | description | appears in changelog |
+| --- | --- | --- |
+| `feat` | A new feature | ✅ |
+| `fix` | A bug fix | ✅ |
+| `perf` | A code change that improves performance | ✅ |
+| `test` | Adding missing tests or correcting existing tests | ❌ |
+| `docs` | Documentation only changes | ❌ |
+| `refactor` | A behavior-neutral code change that neither fixes a bug nor adds a feature | ❌ |
+| `build` | Changes that affect the build system or external dependencies (TypeScript, Jest, pnpm, etc.) | ❌ |
+| `ci` | Changes to CI configuration files and scripts (e.g. Github actions) | ❌ |
```
Comment (tomi) [inline .github/pull_request_title_conventions.md:43]: 😍

File: .github/pull_request_title_conventions.md [hunk 2]
```diff
@@ -55,7 +58,7 @@ The scope should specify the place of the commit change as long as the commit cl
   - n8n → n8n Node
 - `benchmark` - changes to the Benchmark cli
```
Comment (tomi) [inline .github/pull_request_title_conventions.md:59]: Remove the duplicate

**Review by tomi — CHANGES_REQUESTED** [review]

**Review by tomi — APPROVED** [review]
🚀

**Comment by janober** [comment]
Got released with [`n8n@1.58.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.58.0)

**Pull request merged**